src/player.h

```diff
@@ -5,6 +5,7 @@
 #include "item.h"
 #include "player_activity.h"
 #include "weighted_list.h"
+#include "morale.h"
```

Comment:
Why do you need that? I don't see any changes in this header that require including morale.h.